packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -61,19 +68,24 @@ export class VaTelephone {
       formattedNum = `${area}-${local}-${last4}`;
       if (international) formattedNum = `+1-${formattedNum}`;
       if (extension) formattedNum = `${formattedNum}, ext. ${extension}`;
+      if (vanity) {
+        formattedNum = `${area}-${local}-${vanity} (${last4})`;
+      }
     }
     return formattedNum;
   }
 
   /**
    * Format telephone number for screen readers
    * @param {string} number - Expected a formatted phone number with or without extension
-   * @return {string} - Combined phone number parts within the label seperated by
+   * @return {string} - Combined phone number parts within the label separated by
    * periods, e.g. "800-555-1212" becomes "8 0 0. 5 5 5. 1 2 1 2"
    */
-  static formatTelLabel(number: string): string {
-    return number
-      .split(/[^\d\w]+/)
+  static formatTelLabel(number: string, vanity: string): string {
+    const numberType = vanity
+      ? number.split(/[^\d]+/)
+      : number.split(/[^\d\w]+/);
+    return numberType
```

Comment:
Can we add the `vanity` param to the JSDoc above? It might be nice to have a short explanation why we're taking a string param but treating it like a boolean. I was a bit confused when I saw the string param only be used in the conditional part of a ternary, but then I realized that it's because we're passing in the string prop as the arg.

Reply:
@bkjohnson updated and added context into why/how we are using vanity here